app/controllers/appointment_controller.rb

```diff
@@ -0,0 +1,20 @@
+require 'date'
+class AppointmentController < ApplicationController
+    def create_appointment
```

Comment:
This should just be create
```suggestion
    def create
```